model/Software/Classes/PackageVerificationCode.md

```diff
@@ -0,0 +1,53 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# PackageVerificationCode
+
+## Summary
+
+An SPDX version 2.X compatible verification method for software packages.
+
+## Description
+
+This verification method is provided for compatibility with SPDX 2.X.  
+The PackageGitoid is the reccomended verification method for software packages not requiring compatibility with SPDX 2.X.
```

Comment:
I think that gitoid is the best option, and I would instead also keep the option open, to just use the hash of the artifact (as probably the easiest option).

(gitoid is fragile regarding no significant changes like changing an executable flag of a file, that is better in the PackageVerificationCode. Also exclusion of files is a relevant feature, which might make gitoid unusable in some cases)

Reply:
@maxhbr - Can you propose some specific wording to represent your proposed approach?

Reply:
just drop that line? The line above is sufficient

Reply:
Done